packages/core/admin/admin/src/components/RBACProvider/index.js

```diff
@@ -31,7 +31,7 @@ const RBACProvider = ({ children, permissions, refetchPermissions }) => {
 };
 
 RBACProvider.propTypes = {
-  children: PropTypes.element.isRequired,
+  children: PropTypes.arrayOf(PropTypes.element).isRequired,
```

Comment:
Would `PropTypes.node` be more appropriate here? The children aren't limited to React Elements and can be any kind of valid children.

I am not sure, but I think `node` would also cover single and multiple children.